packages/react-examples/src/react-checkbox/Checkbox/Checkbox.Other.Example.tsx

```diff
@@ -38,7 +38,7 @@ function _renderLabelWithLink() {
   return (
     <span>
       Custom-rendered label with a{' '}
-      <Link href="https://www.microsoft.com" target="_blank">
+      <Link href="https://www.microsoft.com" aria-label="go to Microsoft home page" target="_blank">
```

Comment:
I'm not clear on what the problem is here? If it's the fact that the link text is non-descriptive, let's update the text instead of adding an aria-label.

Reply:
The reported issue mentioned that the `Screen reader should announce like as ‘link go to Microsoft home page’`. Changing the text would not result in this exact behavior so, to avoid unwanted changes, an aria-label seemed like the best solution.

Reply:
In this case that wording is just a suggestion, not a requirement to implement that precise behavior. The actual requirement is that "The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context" -- and updating the link text itself to be descriptive is a better way to satisfy that requirement.

Reply:
The other reason for changing the text itself is that in general it's a good practice to have the link text itself be descriptive (it's more helpful for all users that way), and making our examples demonstrate best practices is typically a good thing.

On the other hand...you could argue that the *actual* purpose of this link is just literally the fact that it's a sample link inside a label, so its text doesn't matter. @JustSlone actually commented about this on the original issue as well 😆 
> The link here is not demonstrative of anything in the same and actually the text "link" is perfectly descriptive of the link, since its purpose is to be merely a link.

Reply:
Good point! I've changed the aria-label to the actual text of the link for better clarity.